tests/unit/test_indexer_utils.py

```diff
@@ -0,0 +1,60 @@
+from src.index_runner.es_indexers import indexer_utils
+
+
+def test_merge_default_fields():
```

Comment:
I don't think we need it for this PR, but instead of having real world data, we just need some small test functions that cover each case:

* merging two different scalars makes a list of two elems
* merging two of the same scalar makes a single scalar
* merging two lists creates a union of the elems with no repeats
* setting a default with nothing in the original adds the new key/val
* passing in indexer_ret with no 'doc' throws an error